CONTRIBUTING.md

```diff
@@ -642,14 +642,16 @@ class MyPluginExample1(WebvizPluginABC):
     def __init__(self, arg1: str, arg2: int, arg3: Optional[int] = None):
         ...
 
+def check_deprecation_example2(arg1: int, arg3: int) -> Optional[Tuple[str, str]]:
+    if arg3 == arg1:
+        return ("This message is shown to the end user in the app.", "This message is shown in the documentation of the plugin.")
+    return None
+
 class MyPluginExample2(WebvizPluginABC):
     ...
-    @deprecated_plugin_arguments(check_deprecation)
-    def __init__(self, arg1: str, arg2: int, arg3: Optional[int] = None):
+    @deprecated_plugin_arguments(check_deprecation_example2)
+    def __init__(self, arg1: int, arg2: int, arg3: int):
         ...
 
-def check_deprecation(arg1: int, arg3: int) -> Optional[Tuple[str, str]]:
-    if arg3 == arg1:
-        return ("This message is shown to the end user in the app.", "This message is shown in the documentation of the plugin.")
-    return None
+
```

Comment:
Necessary empty line?

Reply:
Probably just a consequence of me cutting the code block and pasting it further up, leaving the empty line. I can remove it